### ISSUE TITLE
manual-bind-to-arrow does not clean up constructor binds when the same bind occurs multiple times

### ISSUE THREAD

**Issue by seth-outreach** [reactjs/react-codemod#181]
If a file contains multiple classes and some of them define `this.method = this.method.bind(this)`  in their consturctors for methods with identical names, only the first such bind in the file will be removed when applying `manual-bind-to-arrow`

Example input:
```
class FirstOne {
  constructor(...args) {
    super(...args);
    this.aCommonFunctionName = this.aCommonFunctionName.bind(this);
  }

  aCommonFunctionName() {
    return 'a';
  }
}

class SecondOne {
  constructor(...args) {
    super(...args);
    this.aCommonFunctionName = this.aCommonFunctionName.bind(this);
  }

  aCommonFunctionName() {
    return 'b';
  }
}
```

Example output:
```
class FirstOne {
  aCommonFunctionName = () => {
    return 'a';
  };
}

class SecondOne {
  constructor(...args) {
    super(...args);
    this.aCommonFunctionName = this.aCommonFunctionName.bind(this);
  }

  aCommonFunctionName = () => {
    return 'b';
  };
}
```

**Comment by threepointone** [comment]
I'll accept a PR with a fix

**Comment by vivek12345** [comment]
Is there a reason we have this condition in place here
```
if (methods.size() === 0) {
```

Any example where this is useful?

**Comment by threepointone** [comment]
cc @pksjce

**Comment by threepointone** [comment]
> Is there a reason we have this condition in place here
> 
> ```
> if (methods.size() === 0) {
> ```
> 
> Any example where this is useful?

I dunno what you're referring to. If it's code in the transform, I'm not familiar with it. where is it ?

(PS - I'm sorry about this, but this issue was already claimed by @pksjce. If she doesn't want to do it I'd recommend you spend time on it)

**Comment by vivek12345** [comment]
No worries, I understand 👍

**Comment by pksjce** [comment]
Well 🤷‍♀ I did start working on this.

**Comment by threepointone** [comment]
It’s yours!

**Comment by threepointone** [comment]
Fixed by #247. Thank you @pksjce!
